### ISSUE TITLE
MatPES + OMat

### ISSUE THREAD

**Issue by mshuaibii** [materialsvirtuallab/matgl#549]
Congrats on the recent work! I love the idea of finding more systematic subsplits of data for the community to use.

I am rather surprised with the OMat results presented in the paper. Can you clarify how models trained on OMat were evaluated? This caption suggests train/val/test were coming exclusively from their respective datasets:

>  The training, validation, and test sets are randomly selected from the complete
dataset in proportions of 90%:5%:5%

However, a later description then mentions the test set is constructed as follows. Given the numbers, this is what I assume is happening.

> the test set, which comprises 21,737 structures (5%) randomly sampled from
the entire MatPES dataset.

In this scenario, were models directly trained on OMat and then asked to predict MatPES theory? Was there some sort of correction applied to do so? We have seen significant errors if correcting for the level of theory is not performed, even if the VASP settings appear to be close enough. We highlight this in Section 2.3 of the OMat paper - https://arxiv.org/pdf/2410.12771. One way to address this is to use exactly the OMat settings on the MatPES test set and evaluate OMat models against that to ensure consistency. I suspect the errors to go down considerably then.

**Comment by shyuep** [comment]
Thanks. We are indeed rerunning the test set with the exact Omat24 settings. We are also rerunning the formation energies with the exact MatPES settings, just to make sure we have consistency in each of the energy metrics. One important thing of the MatPES work is that we are trying to update the VASP PSPs and settings to the most up to date. There is no single test data that is consistent across all the different models for this reason. We will update the test errors when the calculations are done (not expected to take more than a few days).

I should also point out that the test PES errors are not the most important metrics in the paper. The subsequent MatCalc benchmarks on the structure relaxations, mechanical and phonon properties and MD properties are far more important (and what we focus on). That is why the website also just reports the benchmarks and not the PES test errors. In most of those properties, the difference in performance between Omat24 models and MatPES models is small. The only exception is MD stability, where the Omat24 models are worse (and this is one metric where the DFT settings of the training data have no effect).

**Comment by mshuaibii** [comment]
Sounds great! Definitely, updating the VASP settings is a great direction with MatPES. It just makes comparing across datasets and models less straightforward.

This particular metric just stuck out, the suite of MatCalc are indeed more interesting.
